### DISCUSSION TITLE
Scenarios or Scenario Outline iterations  limit

### DISCUSSION THREAD

**Discussion by ktraditya** [MicroFocus/bdd2octane#29] [Q&A] [answered]
We get this error while pushing our results to octane .
"• There are too many Scenarios or Scenario Outline iterations (36) in this BDD specification. ALM Octane currently supports injecting no more than 30 Scenarios or Scenario Outline iterations in a single BDD specification. "

is this the limitation of the bdd2octane plugin or the octane itself ? can we increase this count ? currently we have lot of feature files that more than 30 scenarios or scenario outline iterations .

**Comment by nissimshitrit** [comment, marked as answer]
Hello,
This limitation is comes from Octane side and you can control on the limit by this site/space parameter 'BDD_SCENARIOS_PER_SPEC_INJECTION_LIMIT'
Just take into account that increasing it may effect on the overall prefomnece
Thanks
